pom.xml

```diff
@@ -586,6 +587,13 @@ under the License.
 				<version>${junit4.version}</version>
 			</dependency>
 
+			<dependency>
+				<groupId>org.assertj</groupId>
+				<artifactId>assertj-core</artifactId>
+				<!-- use 2.9.1 for Java 7 projects -->
```

Comment:
this line doesn't seem useful, surely we aren't going to revert to Java 7.

Reply:
You never know :rofl: